README.md

```diff
@@ -6,7 +6,7 @@ First things first: This is the beginning of a community-driven open-source proj
 
 What `SwiftLog` provides today can be found in the [API docs][api-docs].
 
-## Great, what's the tl;dr
+## Getting started
```

Comment:
Aligning wording style between sswg repos (a bit _too_ casual for a h2 I thought...), sounds good?